packages/canonical-json/package.json

```diff
@@ -29,10 +29,6 @@
   "scripts": {
     "test": "jest"
   },
-  "devDependencies": {
-    "@types/node": "^20.3.3",
-    "typescript": "^5.1.6"
-  },
```

Comment:
Removing common dev dependencies and relying on root `package.json` to keep all subprojects in sync